lti_consumer/lti_consumer.py

```diff
@@ -574,8 +574,8 @@ def lti_provider_key_secret(self):
             try:
                 lti_id, key, secret = [i.strip() for i in lti_passport.split(':')]
             except ValueError:
-                msg = self.ugettext('Could not parse LTI passport: {lti_passport}. Should be "id:key:secret" string.').\
-                    format(lti_passport='{0!r}'.format(lti_passport))
+                msg = 'Could not parse LTI passport: {lti_passport}. Should be "id:key:secret" string.'
+                msg = self.ugettext(msg).format(lti_passport='{0!r}'.format(lti_passport))
```

Comment:
Why can't the following could help? 2 formats doesn't look a right way to do this. 
```
msg = 'Could not parse LTI passport: {lti_passport!r}. Should be "id:key:secret" string'.format(lti_passport=lti_passport)
```